harness/determined/cli/master.py

```diff
@@ -119,6 +124,9 @@ def logs(args: Namespace) -> None:
                 help="number of lines to show, counting from the end "
                 "of the log (default is all)")
         ]),
+        Cmd("cleanup-logs", cleanup_logs, "cleanup expired task logs", [
+            Group(cli.output_format_args["json"], cli.output_format_args["yaml"])
```

Comment:
It may not be necessary to have `Group(cli.output_format_args["json"], cli.output_format_args["yaml"])` as `cleanup_logs()` prints one line.

Reply:
Could we relocate this command to `task.py`, since this is related to task?

Reply:
Removed the yaml/json options. But I think this should stay in master because that's what product wanted